src/v/cloud_storage/remote.cc

```diff
@@ -1268,6 +1268,8 @@ ss::future<upload_result> remote::delete_objects_sequentially(
                              results.push_back(result);
                          });
                    })
+            .handle_exception_type(
+              [](const ss::abort_requested_exception& /* ignore */) {})
             .handle_exception_type([ctxlog](const std::exception& ex) {
```

Comment:
nit: it might not be an issue, but just fyi, check out `ssx::is_shutdown_exception` -- if other kinds of common shutdown-related errors pop up from this call site it might be worth switching over

Reply:
How does this change look?

Reply:
This looks great! Thank you